project.clj

```diff
@@ -44,7 +44,8 @@
             ;;[refactor-nrepl "2.0.0-SNAPSHOT"]
             [lein-ring "0.9.7"]
             [migratus-lein "0.2.6"]
-            [environ/environ.lein "0.2.1"]]
+            [environ/environ.lein "0.2.1"]
+            [lein-environ "1.1.0"]]
```

Comment:
This plugin generates a `.lein-env` file containing the environment variables from the project + profile. I couldn't get the database-url set in the environment without adding it. Is there another way?